home/names.asm

```diff
@@ -7,7 +7,7 @@ NamesPointers:: ; 33ab
 	dbw 0, PartyMonOT
 	dbw 0, OTPartyMonOT
 	dba TrainerClassNames
-	dbw $4, $4b52 ; within PackMenuGFX
+	dbw $4, PackMenuGFX + 60
```

Comment:
Make that `dba PackMenuGFX + 60` since `$4` is `BANK(PackMenuGFX + 60)`.

Reply:
`BANK()` can't evaluate arithmetic (at least I think that's the problem). `dbw BANK(PackMenuGFX), PackMenuGFX + 60` is the best that can be done.

Reply:
Oh well, `dbw BANK(PackMenuGFX), PackMenuGFX + 60` is good enough.